package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-form-validation",
-  "version": "0.0.8",
+  "version": "0.1.0",
   "description": "An helper to make redux form validation easyer",
   "main": "lib/index.js",
```

Comment:
For webpack 2 support you should add `"module": "./src/index.js",` like in `react-router` https://github.com/ReactTraining/react-router/pull/3672/commits/95ad962a0a09524edde78d5fdf387c524b868e5a